### ISSUE TITLE
 detect bank cards and ID cards error

### ISSUE THREAD

**Issue by WenmuZhou** [weiliu89/caffe#991]
Please use the [caffe-users list](https://groups.google.com/forum/#!forum/caffe-users) for usage, installation, or modeling questions, or other requests for help.
_Do not post such requests to Issues._ Doing so interferes with the development of Caffe.

Please read the [guidelines for contributing](https://github.com/BVLC/caffe/blob/master/CONTRIBUTING.md) before submitting this issue.

### Issue summary
Hello, I recently used ssd to detect bank cards and ID cards. I divided the bank card four angles (0, 90, 180, 270), the front four corners of the ID card and the opposite side of the ID card. The data of each angle of the bank card is 25,722. The ID card has 33,724 images per angle.

After training 10 epochs, the output's detection_eval was 0.9151. During the test, both the bank card and the ID card were detected. I also used the validation set of the pascal VOC dataset to test the absence of cards, but these images were detected as bank cards, and the confidence level was 0.7 or more. What might be caused by this situation? I checked the data set and there is no problem.

### Steps to reproduce

If you are having difficulty building Caffe or training a model, please ask the caffe-users mailing list. If you are reporting a build error that seems to be due to a bug in Caffe, please attach your build configuration (either Makefile.config or CMakeCache.txt) and the output of the make (or cmake) command.

### Your system configuration
Operating system:
Compiler:
CUDA version (if applicable):
CUDNN version (if applicable):
BLAS:
Python or MATLAB version (for pycaffe and matcaffe respectively):
